css/footer.css

```diff
@@ -0,0 +1,42 @@
+/* FOOTER STYLING */
```

Comment:
really nice that you've separated the CSS into these separate files! It makes it easier to know where to find styles and also means each page only imports what is required!